docs/installing-northstar/northstar-installers/vtol-guide.md

```diff
@@ -0,0 +1,121 @@
+
+## Skins
+
+![vtol-skins](../../images/vtol-skins.png)
+
+The third settings tab on VTOL is the "Skins" tab. VTOL has support for skins formatted for an older version of Northstar, normally requiring an extra tool. All you need to do is drag the folder of one of these old style skins onto the Skins screen to install it automatically.
+
+## Server
+
+![vtol-server](../../images/vtol-server.png)
+
+The fourth tab on VTOL is the "Server" tab. This section allows you to add Launch Arguments to your client or dedicated server. Note that this adds Launch Arguments to change what Northstar does when it launches, not to launch Northstar itself. A way you can do this is by adding [Launch Arguments](../troubleshooting.md#launch-opts) via the store you own the game on and launching through there.
+
+For the dedicated server option, you can scroll through this tab to set many different options for a server which can be found on the Northstar Wiki's [Server Hosting Guide](../../hosting-a-server-with-northstar/basic-listen-server.md)
+
+## Tools
+
+![vtol-tools](../../images/vtol-tools.png)
+The fifth tab called "Tools" is meant to help those trying to create mods for Northsar. 
+
+## ThunderStore Packer
+
+![vtol-thunderstore-packer](../../images/vtol-thunderstore-packer.png)
+The first screen is an area where you can locate your mod folder, and easily set up things such as the Name, Description, and Icon to be shown on the Thunderstore page when you go to upload your mod.
+
+## Skin Tool
+
+![vtol-skintool](../../images/vtol-skintool.png)
+The second screen is an area where you can drag and drop images to the different maps of a skin to make your own skin. This is an implementaion of the [skin tool](https://github.com/zxcPandora/Titanfall2-SkinTool).
+
+## Advocate
+
+![vtol-advocate](../../images/vtol-advocate.png)
+The third screen is an implementation of Spoon's [Advocate Tool](https://github.com/ASpoonPlaysGames/Advocate), where you can convert an old style skin mod to a mod that can be installed by everyone like a normal mod. Please give credit to the original skin owner while doing this. (If confused, following the earlier link to Github describes how to set up Advocate)
+
+## External Tools
+
+![vtol-external-tools](../../images/vtol-external-tools.png)
+The fourth screen lists a lot of commonly used tools by modders to help create mods easier. You can get more info about each tool by clicking the information icon in the top right of the image for each respective tool.
+
+## About
+
+![vtol-about](../../images/vtol-about.png)
+
+The last section of the VTOL client is the 6th tab of VTOL, the "About" section. This screen shows you what VTOL can do, as well as a button to configure updates, and a button to check for updates for the VTOL client.
+
+# Credits 
+
+[Juicy](https://github.com/BigSpice) for creating and upkeeping the VTOL Manager, as well as helping explain some things that I hadn't seen before
```

Comment:
Note sure if credits are the right thing to have for a wiki. I can add him as co-author on merge if you wanna ^^